### PR TITLE
Add Pass/Fail functionality to NodeVertical

### PR DESCRIPTION
**Pull request by akrzos**
Current checks if cluster loader exits correctly

### CODE REVIEW

**Comment by akrzos** [comment]
DNM this is a WIP for right now however please examine and provide feedback if you have such.

**Comment by akrzos** [comment]
This is dependent upon this functionality being added to cluster loader - https://github.com/openshift/origin/pull/23310

Otherwise we can have situations where NodeVertical workload will hang on a cluster.  The timeout will prevent those situations.

File: docs/nodevertical.md [hunk 1]
```diff
@@ -35,21 +35,33 @@ Default: `quay.io/openshift-scale/scale-ci-workload`
...
 ### WORKLOAD_JOB_PRIVILEGED
-Default: `true`  
+Default: `false`  
 Enables/disables running the workload pod as privileged.
 
 ### KUBECONFIG_FILE
 Default: `~/.kube/config`  
 Location of kubeconfig on orchestration host.
 
+### PBENCH_INSTRUMENTATION
```
Comment (chaitanyaenr) [inline docs/nodevertical.md:53]: The docs of other workloads needs to be modified with the new vars ( PBENCH_INSTRUMENTATION and ENABLE_PBENCH_COPY ). Is the plan to create a PR later?

**Comment by chaitanyaenr** [comment]
Looks good, will give it a try on test cluster and we can merge this.

File: docs/nodevertical.md (hunk 1 above)
Comment (akrzos) [reply docs/nodevertical.md:53]: Eventually yes however those workloads do not implement these vars in the same fashion yet.  I am a bit conflicted on structuring the pbench_* vars as there are a few combinations we will want. (`PBENCH_INSTRUMENTATION`, `ENABLE_PBENCH_AGENTS`, and `ENABLE_PBENCH_COPY`)  For now lets keep this with the nodevertical workload and I will see about extending it to the podvert and mastervert workloads as well.  It is a little bit unclear how we can run with the network tests but I will explore this more once we land this.

The other thought is setting defaults to not require pbench and to be more or less for CI runs.  Thanks for reviewing this.

**Review by chaitanyaenr — APPROVED** [review]
Tested it, LGTM.

File: docs/nodevertical.md (hunk 1 above)
Comment (chaitanyaenr) [reply docs/nodevertical.md:53]: Got it.

**Pull request merged**